Documentation/faq.md

```diff
@@ -1,5 +1,49 @@
 ## Frequently Asked Questions (FAQ)
 
+### etcd, Distributed Systems
+
+#### Failure tolerance in etcd
+
+etcd cluster continues to operate as long as quorum of cluster can communicate with each other. etcd automatically handles transient failures (e.g. network partitions and delays) while ensuring the safety properties in Raft. etcd can also survive hardware failures (e.g. power outages) by persisting Write-Ahead-Log files onto disk, which contain all stream of events that happened. User just needs to restart the node with same configuration, and etcd will automatically replay its log to the exact point of time before failure.
+
+
+### etcd
+
+#### Benchmark etcd? Test workloads?
```

Comment:
add a new section called performance. move this down to the performance section.